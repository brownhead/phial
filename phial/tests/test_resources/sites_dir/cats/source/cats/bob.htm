name: Bob Dole
...

<div>Meow Dole</div>
