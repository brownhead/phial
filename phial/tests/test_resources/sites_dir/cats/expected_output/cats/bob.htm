<div>
	<a href="../index.htm">Go Home!</a>
</div>
<h1>Bob Dole</h1>
<div>
	
<div>
	My owner is not creative.
</div>


</div>
